exercises/rna-transcription/source/rnatranscription/RNAtest.ceylon

```diff
@@ -0,0 +1,31 @@
+import ceylon.test { ... }
+
+// Tests adapted from x-common version 1.0.0
+{[String, String|Error]*} cases => {
+  // empty strand
```

Comment:
I accumulated tests from both of the Scheme and Java tracks (though I can't remember which this one came from; Java, I think). I can open up a pull request to add that test.